sys/include/net/gnrc/udp.h

```diff
@@ -54,6 +54,22 @@ extern "C" {
 #endif
 
 /**
+ * @brief   Default UDP port range for dynamic source port allocation
+ */
+#define GNRC_UDP_DYN_PORTRANGE_MIN (49152U)
+#define GNRC_UDP_DYN_PORTRANGE_MAX (65535U)
+#define GNRC_UDP_DYN_PORTRANGE_NUM (GNRC_UDP_DYN_PORTRANGE_MAX - GNRC_UDP_DYN_PORTRANGE_MIN + 1)
+#define GNRC_UDP_DYN_PORTRANGE_ERR (0)
+
+/**
+ * @brief   Offset for next dynamic UDP port
+ *
+ * Currently set to a static (prime) offset, but could be random, too
+ * see https://tools.ietf.org/html/rfc6056#section-3.3.3
+ */
+#define GNRC_UDP_DYN_PORTRANGE_OFF (17U)
+
+/**
```

Comment:
Mhhhh sleeping over it a bit: why are these in `gnrc/udp.h`? Since they are not really used by `gnrc_udp` directly this can be confusing. I would vote putting them either in `gnrc_sock`'s internal header or directly into the c-file.